appveyor/dev/law-xml/appveyor.yml

```
@@ -110,7 +89,12 @@ before_build:
   - git -C ..\%UPSTREAM% rev-parse HEAD
 
 build_script:
-  - ps: $env:FLAG = if ($env:APPVEYOR_PULL_REQUEST_NUMBER -eq $null) { "--deploy" }
+  - ps: >-
+      If ($env:APPVEYOR_SCHEDULED_BUILD -eq $True){
+        $run_sync = $env:PYTHON + '\python.exe -m oll.tools.cli ci synch-xml-with-upstream --library-root .. --auth ../law --upstream-url ' + $env:UPSTREAM_XML + ' --upstream-auth-url ' + $env:UPSTREAM_AUTH;
```

We already have an `UPSTREAM` variable which is missing `https://github.com/` prefix. So I think we can simplify and remove `UPSTREAM_XML` variable . The end command looks something like this:

```suggestion
        $run_sync = $env:PYTHON + '\python.exe -m oll.tools.cli ci synch-xml-with-upstream --library-root .. --auth ../law --upstream-url ' + https://github.com/$env:UPSTREAM + ' --upstream-auth-url ' + $env:UPSTREAM_AUTH;
```
I think we'll have to keep `UPSTREAM_AUTH` though.

---

I noticed that, but was thinking that it might be better to avoid hardcoding github.com in such a way (alternatively, `UPSTREAM_AUTH` can also just contain the name, without the github part so that it's all consistent)